tests/transformation_test.py

```diff
@@ -258,6 +258,18 @@ def test_pdf_convert(transform):
     result = transform.pdf_convert(pageorientation='landscape')
     assert result.url == target_url
 
+def test_minify_css(transform):
+    target_url = '{}/{}/minify_css/{}'.format(config.CDN_URL, APIKEY, EXTERNAL_URL)
+    result = transform.minify_css()
+    assert result.url == target_url
+
+
+def test_minify_css_with_params(transform):
+    target_url = '{}/{}/minify_css=gzip:fals    e,level:1/{}'.format(config.CDN_URL, APIKEY, EXTERNAL_URL)
+    result = transform.minify_css(level=1,gzip=False)
```

Comment:
And one more missed space between parameters.

Reply:
fixed